### PR TITLE
Full-Stack-Capstone-Backend

### PR DESCRIPTION
**Pull request by bhushan354**
Hello reviewer , 
When we started this Project we were group of 5 partners but now we are group of three partners so we have implemented ( requirements.

Link to front-end PR : https://github.com/bhushan354/Full-Stack-Capstone-Frontend/pull/21

Project Requirements : -

1. The user logs in to the website, only by typing the username (a proper authenticated login is a requirement if group is made of 5 people).
2. In the navigation panel, the user can see links to:
   - Motorcycles/doctors/classes/items that you selected as a theme.
   - "Reserve" form.
   - "My reservations".
   - "Add motorcycle/doctor/class/item that you selected as a theme" (the link is visible to everybody).
   - "Delete motorcycle/doctor/class/item that you selected as a theme" (the link is visible to everybody).
3. On the main page, the user can see a Vehicles.
4. When the user selects a specific item, they can see the details page with its full description (skip the "Rotate image" button).

   - In the details page, the user can click the "Reserve" button .

5. When the user clicks the "Add item" link in the navigation panel they can see a form for adding a new item.

6. Make the app responsive, creating both mobile and desktop versions.

7. When the user clicks the "Delete item" link in the navigation panel they can see a list of all items with title and "Delete" button.
   - When the user clicks the "Delete" button, the selected item is marked as removed and does not show on the main list anymore.

- To reserve an appointment, the user has to select a date and city (username and selected item are autofilled).

8. When the user clicks the "My reservations" link in the navigation panel they can see a list of their reservations (with information about item name, date and city).
9. Add full documentation for your API.

Thank you ,
Bhushan , Umair, Eugene .

### CODE REVIEW

**Comment by bhushan354** [comment]
Hello air ,

if you found any bugs then please give a hint about how we can solve that particular error . it will be very helpful as we have very little time for project deadline .

Thank you !
Have a good day .

**Comment by bhushan354** [comment]
Hello Sir/Mam ,

if you found any bugs then please give a hint about how we can solve that particular error . it will be very helpful as we have very little time for project deadline .

Thank you !
Have a good day .

File: config/database.yml [hunk 1]
```diff
@@ -1,84 +1,19 @@
...
-  #min_messages: notice
-
-# Warning: The database defined as "test" will be erased and
-# re-generated from your development database when you run "rake".
-# Do not set this db to the same as development or production.
+  database: booking_cap_db
 test:
   <<: *default
-  database: full_stack_capstone_backend_test
+  database: booking_cap_db
```
Comment (Meltrust) [inline config/database.yml]: - [ ] The testing database and the dev database have the same name, resulting on this:

image

Kindly, change the name of the testing database, so the name is different than the dev database.  You could use a name like `booking_cap_db_test`. 👍

Comment (Meltrust) [inline config/database.yml]: - [ ]  You are using rswag, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

File: README.md [hunk 2]
```diff
@@ -1,24 +1,183 @@
...
+- Clone this [repository](git@github.com:bhushan354/Full-Stack-Capstone-Backend.git) to your desired folder:
+
+- Run this command in your command line interface:
+
+```sh
+  cd [YOUR FOLDER]
+  git git@github.com:bhushan354/Full-Stack-Capstone-Backend.git
+  cd Full-Stack-Capstone-Backend
+  rails db:migrate
+  rails db:seed
```
Comment (Meltrust) [inline README.md]: - [ ] Before migrating and seeding the database, please include the command `rails db:create` so your users are able to create the database. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!










#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

**Comment by bhushan354** [comment]
Hello Dear reviewer, 
We have updated the Readme, if you follow the README instructions carefully you will get a running app on your localmachine and still if you get stuck anywhere you can reach out to us on zoom or slack .

Thank you ,
Bhushan , Umair , Eugine .

**Review by vichuge — COMMENTED** [review]
Hi dear student, 

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

InbornCandidHornet-size_restricted

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**